src/main/docs/guide/openApiViews/redoc.adoc

```diff
@@ -36,17 +36,17 @@ See https://github.com/Redocly/redoc#redoc-options-object[Redoc Options] for a d
 To expose the `redoc` views, you also must expose the generated `yaml`:
 
 .Exposing Swagger YAML And Redoc Views
-[configuration]
+[source,yaml]
```

Comment:
we want to use the `configuration` macro

Reply:
fixed